package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "knex-mysql-deadlock",
+  "version": "1.0.0",
+  "description": "A custom knex mysql dialect, which does deadlock retries",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Mindflash/knex-mysql-deadlock.git"
+  },
+  "author": "mindflash",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Mindflash/knex-mysql-deadlock/issues"
+  },
+  "homepage": "https://github.com/Mindflash/knex-mysql-deadlock#readme",
+  "dependencies": {
+    "knex": "^0.12.6"
```

Comment:
I agree on knex, will make it a peerDep and will update README accordingly. For node-mysql - still disagree. It is up to knex which driver library to use. They might switch from node-mysql to node-bla-mysql, and we have 0 control on that.